src/Xablu.WebApiClient/WebApiClient.cs

```diff
@@ -144,28 +144,29 @@ public async Task<TModel> SendMutationAsync<TModel>(Request<TModel> request, Pri
                 message += string.Join(" /// ", result.Errors.Select(e => e.Message));
                 throw new Exception(message);
             }
-
-            var resultData = result?.Data as JObject;
+            var name = typeof(TModel).Name;
+            var variableName = char.ToLower(name[0]) + name?.Substring(1);
+            var resultData = result?.GetDataFieldAs<TModel>(variableName);
```

Comment:
This doesn't work for BooksQL, this is what I receive:

![imagen](https://user-images.githubusercontent.com/12127846/72897678-42035e00-3d01-11ea-8102-6e1f9b43f4f6.png)

Where `TModel` is of type `BookReview`, so the generated string `bookReview` doesn't match the returned operation `createReview`.

Please make sure you test this in more than a single scenario, using the BooksQL example is an option.

Reply:
fixed data object contains a returning operation name. It now correctly maps to an Object and returns this.